Mathlib/Algebra/Algebra/Bilinear.lean

```diff
@@ -223,13 +223,13 @@ theorem mulRight_eq_zero_iff (a : A) : mulRight R a = 0 ↔ a = 0 := by
 @[simp]
 theorem mulLeft_one : mulLeft R (1 : A) = LinearMap.id := by
   ext
-  simp only [LinearMap.id_coe, one_mul, id.def, mulLeft_apply]
+  simp
```

Comment:
Is this change intended or needed? A priori, doesn't seem like an improvement to me.

Reply:
I made the change because on nightly-testing it needs changing as written, but if we just do the simpler thing of trusting `simp` to close the goal, the proof works as written on master and on nightly-testing.

`simp only` is not necessarily more robust!